build.gradle.kts

```diff
@@ -73,84 +62,11 @@ java {
     targetCompatibility = JavaVersion.VERSION_1_8
 }
 
-semver {
-    tagPrefix("v")
-    initialVersion("0.1.0")
-    findProperty("semver.overrideVersion")?.toString()?.let { overrideVersion(it) }
-    val semVerModifier = findProperty("semver.modifier")?.toString()?.let { buildVersionModifier(it) } ?: { nextPatch() }
-    versionModifier(semVerModifier)
-}
-
-/**
- * The code below is a workaround for [gradle/gradle#20016](https://github.com/gradle/gradle/issues/20016) derived from
- * [platform-portal@14319ed/docs/semver.md](https://github.com/FigureTechnologies/platform-portal/blob/14319ed7e97d88a1b8cbb2f2f7708cc0660dc518/docs/semver.md#limiting-dependencies-to-release-versions)
- * to prevent version strings like `"1.0.+"` from resolving to pre-release versions.
- * You can adjust the string set constants or comment the code out entirely to allow pre-release versions to be used.
- */
```

Comment:
I didn't realize we had this snippet in this service. It could still be useful to have, but it's probably not needed at the moment, so I'm fine with removing it now.

Reply:
Ill keep an eye out for any issues that crop up from this. hopefully we can get away with making the build much easier to read.